docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -391,16 +391,25 @@ end
 | Pending
 | Yes
 | Yes
-| 2.11.0
-| -
+| 2.11
+| 2.13
 |===
 
 Prefer negated matchers over `to change.by(0)`.
 
-This cop does not support autocorrection in some cases.
+In the case of composite expectations, cop suggest using the
+negation matchers of `RSpec::Matchers#change`.
+
+By default the cop does not support autocorrect of
+compound expectations, but if you set the
+negated matcher for `change`, e.g. `not_change` with
+the `NegatedMatcher` option, the cop will perform the autocorrection.
+`NegatedMatcher` option, will be autocorrect.
```

Comment:
this seems redundant?

Reply:
I was adding that I thought it might be confusing for cases that does not support autocorrected. I think a polite explanation would be about right. What do you think?

Reply:
I mean that the last sentence, ``NegatedMatcher` option, will be autocorrect.` is just repeating the previous one, and it's also incomplete anyway

Reply:
Ah! I was mistaken. Thank you very much. It is indeed so I have corrected it. How about it?